### PR TITLE
[PTQ][OV] Check out empty statistics

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- The logic of TensorStatistic has been updated to ensure proper functioning even in the absence of any statistics.

### Reason for changes

- The error during the quantization

### Related tickets

- 107080

### CODE REVIEW

File: nncf/experimental/common/tensor_statistics/collectors.py [hunk 1]
```diff
@@ -402,23 +403,31 @@ def _reduce_out_of_place(self, x: List[TensorType]) -> List[TensorType]:
 
 class MinReducer(TensorReducerBase):
     def _reduce_out_of_place(self, x: List[NNCFTensor]) -> List[TensorType]:
-        return [self._tensor_processor.reduce_min(x[0], self._reduction_shape, keepdims=True)]
+        t = x[0]
+        fn = self._tensor_processor.reduce_min
+        return [None if t.is_empty() else fn(t, self._reduction_shape, keepdims=True)]
```
[the remark this answers is not in the document]
Comment (daniil-lyakhov) [reply nncf/experimental/common/tensor_statistics/collectors.py:408]: Yes, but what if `MinReduer` is in inplace mode and its statistic point is empty? Will this check https://github.com/openvinotoolkit/nncf/pull/1728/files#diff-6e8ff64441fe5f0da52a90b8958f05b9525f20e57e9e85916a3d0193bb495f1dR142 wokr?

[the remark this answers is not in the document]
Comment (KodiaqQ) [reply nncf/experimental/common/tensor_statistics/collectors.py]: Done with test

**Comment by KodiaqQ** [comment]
> Please, implement the check in the base class by offline agreement.

Done.

File: nncf/experimental/common/tensor_statistics/collectors.py [hunk 2]
```diff
@@ -448,14 +451,14 @@ class BatchMeanReducer(TensorReducerBase):
     def __init__(self, inplace: bool = False):
         super().__init__(None, inplace)
 
-    def _reduce_out_of_place(self, x: List[NNCFTensor]) -> List[TensorType]:
+    def _reduce_out_of_place(self, x: List[NNCFTensor]) -> List[NNCFTensor]:
```
Comment (daniil-lyakhov) [inline nncf/experimental/common/tensor_statistics/collectors.py:454]: Thanks 👍

**Review by daniil-lyakhov — APPROVED** [review]

**Review by l-bat — APPROVED** [review]

**Review by alexsu52 — APPROVED** [review]
LGTM

**Pull request merged**